README.md

```diff
@@ -1,27 +1,22 @@
-Battle for the Net Widget
--------------------------
-#### Join the battle for net neutrality with our action widget!
+# How to install the widget
 
-September 10th, 2014 is [**the Internet Slowdown!**][1] The goal of this
-project is to allow anyone with a web site to run their own campaign to save net
-neutrality. Simply embed a couple of lines of Javascript and you're good to go!
+Add this to any page, and you're golden: ([**See the demo!**](https://widget.battleforthenet.com/demos/modal.html))
 
-To stay up-to-date on developments, please subscribe to our
-[**developer mailing list**][1]. If you have any problems or questions regarding
-the widget, please [submit an issue][4] or [email Jeff Lyon][5].
+```html
+<script src="https://widget.battleforthenet.com/widget.js" async></script>
+```
+
+The goal of this project is to allow anyone with a web site to run their own campaign to save net neutrality. Simply add one line of Javascript and you're good to go! The modal animation will show up front-and-center on your page, prompting
+visitors to contact Congress and the FCC.
 
+If you have any problems or questions regarding the widget, please [submit an issue](https://github.com/fightforthefuture/battleforthenet-widget/issues).
 
-How it works
-------------
-#### Non-technical version:
 
-* Developer embeds widget code snippet on their site (either modal or banner)
-* On September 10th, 2014, users are shown the widget on their first page view
-* Users can interact with or dismiss the widget
-* A cookie is written so the user doesn't see the widget again
+# How it works
 
-#### Technical version:
-Please take a look at [**widget.js**][6] if you want to see exactly what you'll
+The widget is designed to only appear on July 12, 2017, and only once, per user, per device. If you'd like to force it to show up on your page for testing, please reload the page with `#SHOW_BFTN_WIDGET` at the end of the URL.
```

Comment:
Is "only appear on July 12, 2017" true? I think it is only true if one copies the widget to their own site. If one loads from widget.battleforthenet.com/widget.min.js you'll get whatever code is served. People who used that URL to have an action "only appear on September 10th, 2014" will be having something appear again this July 12. I imagine many people might want to opt in to future actions, but it might make sense to say that's what they're doing by referencing rather than copying the widget code. Maybe I'm missing something, happy to be wrong!